modules/ROOT/pages/dataweave-formats-flatfile.adoc

```diff
@@ -66,8 +66,8 @@ Valid values are `true` or `false`.
 |`recordParsing`  |`String`|`'strict'`|Specifies the expected type of separation between lines or records:
 
 * `strict`: Line break is expected at exact end of each record. `strict` is the default.
-* `lenient`: Line break is used, but records can be shorter or longer than the schema specifies.
-* `noTerminator`: Records follow one another with no separation.
+* `lenient`: Line break is used, but records can be shorter or longer than the schema specifies. Do not use `lenient` if your payload lacks line breaks. The other options to `recordParsing` support records that lack line breaks.
```

Comment:
```suggestion
* `lenient`: Line break is used, but records can be shorter or longer than the schema specifies. Don't use `lenient` if your payload lacks line breaks. The other options to `recordParsing` support records that lack line breaks.
```

Reply:
this stuff will also need to be updated in the source; maybe the devs can help if necessary, or let me know, and I'll help